docs/recipes.md

```
@@ -177,9 +177,53 @@ Returns:
+# Gate with explicit policy file
+tokmd gate --policy policy.toml
+
+# Compute analysis then gate
+tokmd gate --preset health
+
+# JSON output for CI parsing
+tokmd gate --format json
+```
+
+**Example policy.toml**:
+```toml
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)